phase-4-explainer.md

```diff
@@ -0,0 +1,175 @@
+### Phase 4: Full Introspection of an Accessibility Tree
```

Comment:
```
npm install -g doctoc
doctoc phase-4-explainer.md
```

Reply:
Also these heading levels could all be upgraded, this should be a `#` not a `###` etc.

Reply:
Yes, sorry, I edited this online. I'll switch to doing it locally so I can use doctoc.

Reply:
Done.